e2e/pages/Story.js

```diff
@@ -1,29 +1,26 @@
 const Page = require('./Page');
 const StoryTask = require('./StoryTask');
-let CommonActions = require('../core/ui/CommonActions.js');
+const CommonActions = require('../core/ui/CommonActions.js');
 /**
  * this class contains methods of dashboard.
  */
 class Story extends Page {
     constructor() {
         super();
-        this.addStoryButton = 'button[data-aid="Sidebar__AddStoryButton"]';
         this.storyTitleField = 'textArea[aria-label="story title"]';
         this.saveStoryButton = 'button[class="autosaves button std save"]';
     }
     open(projectId) {
         super.open(`/n/projects/${projectId}`);
     }
-    clickAddStoryButton() {
-        CommonActions.waitAndClick(this.addStoryButton);
-    }
     setStoryTitleField(storyTitle) {
         CommonActions.waitAndSetValue(this.storyTitleField,storyTitle);
     }
     clickSaveStoryButton() {
         CommonActions.waitAndClick(this.saveStoryButton);
         return new StoryTask();
     }
+    //TODO change input for a json value
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1135)